src/Entities/CollectionGroup.ts

```diff
@@ -19,6 +19,10 @@ export class CollectionGroup {
   @ObjectIdColumn()
   id: ObjectID
 
+  @Field(type => ID, { nullable: true })
```

Comment:
Why is this nullable? (I see that `id` has no specification at all, but also doesn’t show up in the schema at all?

Reply:
I'm not sure about the original design intent. @pepopowitz, @mbilokonsky, or @xtina-starr do any of you know?

Reply:
Sorry I purely cargo-culted adding the original ID field when I wrote this code. If we don't need it feel free to rip it out, I was just following what I saw from other files I'm not as familiar with this ORM.